docs/content/features/exploiters/mssql.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "MSSQL Exploiter"
+draft: false
+description: "Exploits MSSQL by taking advantage of insecure configuration"
+tags: ["exploiter", "mssql", "brute force"]
+pre: "<i class='fa fa-database'></i> "
+---
+
+## MSSQL
+
+[MSSQL (Microsoft SQL Server)](
+https://learn.microsoft.com/en-us/sql/sql-server/what-is-sql-server) is a
+relational database management system (RDBMS) developed by Microsoft whose
+functions include managing, storing, retrieving, manipulating, and analyzing
+data efficiently.
+
+[`xp_cmdshell`](
+https://learn.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/xp-cmdshell-transact-sql)
+is a procedure in MSSQL that allows users to execute Windows shell commands
+from within the SQL Server environment.
+
+## Exploitation
+
+Machines with MSSQL that have `xp_cmdshell` enabled may be accessible to
+attackers if they come across the correct credentials.
```

Comment:
Maybe:

```suggestion
Machines with MSSQL that have `xp_cmdshell` enabled may be accessible to
attackers if they are able to successfully authenticate with the service.
```

?

Reply:
If they're able to successfully authenticate, the machine ~may~ will be accessible. Seems kind of unnecessary to mention it. Between these two, I prefer the former.  What about it do you not like?

Reply:
How about this?
```suggestion
Machines with MSSQL that have `xp_cmdshell` enabled may be accessible to
attackers if they find a way to authenticate with the service.
```

Reply:
It makes it sound so haphazard. We could just say "if they are able to authenticate with the service."

Reply:
"if they are able to authenticate with the service" suggests they have the right credentials + `xp_cmdshell` was enabled, it feels wrong to say "_may_ be accessible" with that. I'm having trouble thinking of a better sentence.